packages/editor/src/components/rich-text/style.scss

```diff
@@ -94,12 +94,10 @@
 		pointer-events: none;
 
 		// Use opacity to work in various editor styles.
+		// We don't specify the color here, because blocks or editor styles might provide their own.
 		&,
 		p {
-			color: $dark-opacity-300;
-			.is-dark-theme & {
-				color: $light-opacity-300;
-			}
+			opacity: 0.62;
```

Comment:
That's an awfully specific percent 😆

Reply:
Yep ;) 

It's from here: https://github.com/WordPress/gutenberg/blob/master/edit-post/assets/stylesheets/_colors.scss#L36 — which is part of the array of opacity based colors which were very carefully put together in https://github.com/WordPress/gutenberg/pull/6937

In other words, we're using the same opacity as the previous RGBA variable does. I did try rounding up, but that lost a surprising amount of contrast, in both directions.